### DISCUSSION TITLE
Suggesting IDE in the README.md

### DISCUSSION THREAD

**Discussion by tuxy-streamer** [RajeshTechForge/pinac_workspace#84] [General]
I don't think we should be recommending ide or extensions for development.
It seems very rude to me and should be removed.

**Comment by RajeshTechForge** [comment]
A debatable topic!

I recommended the extension because the codebase is formatted by the style of the code formatter extension. And for that IDE comes by default.

I appreciate your opinion.

**Comment by tuxy-streamer** [comment]
You could have easily said to use 2 spaces indentation.
I faced issues cause I normally use 1 tab (4 spaces) indentation.
I know you're tallking about prettier but as far as I know 'ESLint' is a LSP and not a code formatter.
So, why asking to use that ?

**Reply by RajeshTechForge** [reply]
I am not asking you to use that, it's just a recommendation, you can have your choice !
